Sources/COWMacros/SyntaxExtensions.swift

```diff
@@ -129,6 +137,107 @@ extension FunctionDeclSyntax {
         }
       }
     }
+    return false
+  }
+  
+  var isMutating: Bool {
+    modifiers.contains(where: { $0.name.tokenKind == .keyword(.`mutating`) })
+  }
+  
+  func returnTypeEquals(to type: String) -> Bool {
```

Comment:
Explicit `internal` access control is recommended. This convention is learned from the Swift Standard Library.